### ISSUE TITLE
Adding items to queue

### ISSUE THREAD

**Issue by neoassyrian** [react-native-google-cast/react-native-google-cast#233]
Hi, 

Is the queue functionality available in V4 beta yet? 
If it is, can someone help me on how to add the items to the queue.
Do we use "loadMedia" method on the RemoteMediaClient or "queueInsertItems"? 

The "MediaLoadRequest" class expects either a "mediaInfo" or "queueData".
The queueData (MediaQueueData) has an "items" property but this is not an array and it is an "MediaQueueItem" which also doesn't have a property that accepts an array of "MediaInfo" objects. 

Can someone help please.

**Comment by petrbela** [comment]
Good catch! The `items` should be an array of `MediaQueueItem` objects. It's just a TypeScript definition error so you should be able to pass an array there:

```js
client.loadMedia({
  queueData: {
    items: [{
      mediaInfo: {
        contentUrl: 'https://commondatastorage.googleapis.com/gtv-videos-bucket/CastVideos/mp4/BigBuckBunny.mp4',
      }
    }]
  }
})

I'll fix it in the next beta release (hopefully today).

The other methods such as `queueInsertItems` allow you to add more items to an existing queue (`loadMedia` replaces it).

**Comment by neoassyrian** [comment]
Hi @petrbela 
Thanks for getting back to me. That's what I thought and I was trying to supply an array but then I started getting an error. I removed my initial code and supplied an array first to test it, but I'm getting this error:
Possible Unhandled Promise Rejection (id: 0):
Error: java.util.HashMap cannot be cast to com.facebook.react.bridge.ReadableMap
Error: java.util.HashMap cannot be cast to com.facebook.react.bridge.ReadableMap
    at promiseMethodWrapper (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false:4976:45)
    at loadMedia (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false:152516:32)

**Comment by petrbela** [comment]
Hmm, that looks like a bug... I'll check it tomorrow!

**Comment by neoassyrian** [comment]
Thank you @petrbela 
I'm happy to give it a test. I tried to figure out where it is happening but everything looked right. 

Please let me know. Would love to try this feature.

**Comment by petrbela** [comment]
Fixed in 4.0.0-beta.2
